src/ClimaAtmos/Dycore/src/Topologies.jl

```diff
@@ -1,14 +1,117 @@
 module Topologies
 
 export AbstractTopology, AbstractStackedTopology
+export GenericTopology
```

Comment:
Do we need `AbstractTopology` and `GenericTopology`, or should be just have `Topology`

Reply:
Agreed, I removed `AbstractTopology` and `GenericTopology` and replaced them with simply `Topology`.